### PR TITLE
fix: Gutenberg cover block container width CSS cases.

### PR DESCRIPTION
**Pull request by shubhw12**
### Description
Checked the issues reported on the website demos.

### How has this been tested?
Checked by importing the same pages and some demo random sites.

### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: inc/class-astra-dynamic-css.php [hunk 1]
```diff
@@ -687,8 +687,12 @@ public static function return_output( $dynamic_css, $dynamic_css_filtered = '' )
 						'margin-right'  => 'auto',
 						'padding-left'  => '20px',
 						'padding-right' => '20px',
-						'width'         => astra_get_css_value( $site_content_width + 40, 'px' ),
 					),
+
+					'ast-plain-container.ast-no-sidebar .entry-content .alignwide .wp-block-cover__inner-container, .ast-plain-container.ast-no-sidebar .entry-content .alignfull .wp-block-cover__inner-container' => array(
```
Comment (premanshup) [inline inc/class-astra-dynamic-css.php:692]: @shubhw12 `.` is missing in the `ast-plain-container.ast-no-sidebar` CSS selector.

File: inc/class-astra-dynamic-css.php [hunk 2]
```diff
@@ -687,8 +687,12 @@ public static function return_output( $dynamic_css, $dynamic_css_filtered = '' )
...
+						'width' => astra_get_css_value( $site_content_width + 40, 'px' ), 
+					),
+					//
```
Comment (premanshup) [inline inc/class-astra-dynamic-css.php:695]: @shubhw12 Please remove this additional comment.

**Review by premanshup — CHANGES_REQUESTED** [review]

File: inc/class-astra-dynamic-css.php (hunk 1 above)
Comment (shubhw12) [reply inc/class-astra-dynamic-css.php]: Done

File: inc/class-astra-dynamic-css.php (hunk 2 above)
Comment (shubhw12) [reply inc/class-astra-dynamic-css.php]: Removed the comment

**Review by premanshup — APPROVED** [review]

**Pull request merged**